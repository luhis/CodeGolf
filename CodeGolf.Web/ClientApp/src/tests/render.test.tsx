import { h } from "preact";
import render from "preact-render-to-string";
import Header from "../components/header/index";

test("Link changes the class when hovered", () => {
    const component = render(
        <Header />,
    );
    expect(component).toMatchSnapshot();
});
