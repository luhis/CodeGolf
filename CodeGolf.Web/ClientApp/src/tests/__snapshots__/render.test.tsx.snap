// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `"<nav class=\\"navbar\\" role=\\"navigation\\" aria-label=\\"main navigation\\"><div class=\\"navbar-brand\\"><a role=\\"button\\" class=\\"navbar-burger burger \\" aria-label=\\"menu\\" aria-expanded=\\"false\\"><span aria-hidden=\\"true\\"></span><span aria-hidden=\\"true\\"></span><span aria-hidden=\\"true\\"></span></a></div><div class=\\"navbar-menu \\"><div class=\\"navbar-start\\"><a class=\\"navbar-item\\" href=\\"/\\">Home</a><a class=\\"navbar-item\\" href=\\"/demo\\">Demo</a><a href=\\"/account/signin\\" class=\\"navbar-item\\">Sign In with GitHub</a><a class=\\"navbar-item\\" target=\\"_blank\\" rel=\\"noopener\\" href=\\"https://codegolf.stackexchange.com/questions/173/tips-for-code-golfing-in-c\\">Tips</a></div></div></nav>"`;
